code/modules/mining/equipment/survival_pod.dm

```diff
@@ -124,20 +124,20 @@
 	//smooth = SMOOTH_MORE //MONKESTATION REMOVAL
 	//canSmoothWith = list(/turf/closed/wall/mineral/titanium/survival, /obj/machinery/door/airlock/survival_pod, /obj/structure/window/shuttle/survival_pod) //MONKESTATION REMOVAL
 
-/obj/structure/window/shuttle/survival_pod/spawner/north
+/obj/structure/window/reinforced/survival_pod
```

Comment:
Add a /south typepath for ease of locating.

Reply:
> Since you're working on these spawners anyway, can you clean up the hollow versions as well? They only have a couple directions and cannot be rotated.

They're fixed by this, yeah